Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: build build-operator lint vet test integration check clean
+.PHONY: build build-operator lint vet test integration treeshake check clean
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)